shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -91,6 +97,19 @@ public void removeIsDisplayingFlutterUiListener(@NonNull FlutterUiDisplayListene
     flutterJNI.removeIsDisplayingFlutterUiListener(listener);
   }
 
+  /** Adds a listener that is invoked when a memory pressure warning was forward. */
+  public void addOnLowMemoryListener(@NonNull OnLowMemoryListener listener) {
```

Comment:
can we make this private or package-only, and add `VisibleForTesting`>

Reply:
done.